README.md

```diff
@@ -1,17 +1,21 @@
+[![As-Is](https://img.shields.io/badge/Support%20Level-As--Is-e8762c.svg)](https://www.tableau.com/support-levels-it-and-developer-tools)
+
 # Filter Bookmarks
 This extension enables you to add a button directly into your dashboard that resets to certain filter settings. Great for embedded dashboards that do not have the toolbar showing or for a more intuitive interface.
 
-## How to use an Extension
-Download the Filter Bookmarks [manifest file](https://extensiongallery.tableau.com/products/29). Open Tableau Desktop 2018.2 or higher, drag in the "Extension" object to a dashboard. Click "My Extensions" and find the manifest file (.trex) you downloaded above.
+## Installing the Extension
+See the Tableau Help topic [Use Dashboard Extensions](https://help.tableau.com/current/pro/desktop/en-us/dashboard_extensions.htm) for directions. When presented with the list of available Dashboard Extensions, search for Filter Bookmarks to find and install this one.
 
-# Using the Extension
+## Using the Extension
 1. Set your dashboard filters they way you want to be able to revert to.
 2. Open the configuration window and click "Save Settings".
 3. Optional: Customize your buttons label and colors.
 
 Note: You can add as many instances of this extension as you like!
 
-## How to install for local use
+## Building for local use
```

Comment:
Is there any scenario where they want to install it locally and NOT update any code? Or can we make this heading slightly clearer/more offputting like 

1. Using the Extension from the Tableau Gallery (recommended)
a. set your dashboard....


2. Download the Extension code to develop locally
a...

Reply:
Good point - I think it's just the 2 modes, either use from Exchange or DIY. I pushed an update per your wording advice.

Reply:
<img width="866" alt="image" src="https://github.com/tableau/extension-filter-bookmarks/assets/58372/ada1bb0a-da76-4a45-950c-3b7a2841cabe">